src/components/useraccount/orderdetail/MainDetails.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div v-if="me">
+    <div class="row">
+      <div class="col-sm-6">
+        <div class="my-orders-order-dark-box">
+          <div class="my-orders-order-dark-box-title">{{ $t('shippingAddress') }}</div>
+          <div>
+            {{me.order.shippingAddress.title}}
```

Comment:
```suggestion
            {{ me.order.shippingAddress.title }}
```

And the same for all the ones in the rest of the PR ;)